executor/api/rest/server.go

```diff
@@ -153,7 +153,9 @@ func (r *SeldonRestApi) Initialise() {
 
 		case api.ProtocolTensorflow:
 			r.Router.NewRoute().Path("/v1/models/{"+ModelHttpPathVariable+"}/:predict").Methods("OPTIONS", "POST").HandlerFunc(r.wrapMetrics(metric.PredictionHttpServiceName, r.predictions))
+			r.Router.NewRoute().Path("/v1/models/{"+ModelHttpPathVariable+"}:predict").Methods("OPTIONS", "POST").HandlerFunc(r.wrapMetrics(metric.PredictionHttpServiceName, r.predictions))
```

Comment:
May be worth adding a comment on this one as well as it's hard to spot the `/`

Reply:
done